package.json

```diff
@@ -37,9 +37,9 @@
     "@angular/compiler": "^10.0.11",
     "@angular/compiler-cli": "^10.0.11",
     "@angular/core": "^10.0.11",
-    "ng-packagr": "^10.1.0"
+    "ng-packagr": "^10.1.0",
```

Comment:
@ayush013 Can you remove all the carat signs from all the packages?

Reply:
Sure